wagtailnhsukfrontend/templates/wagtailnhsukfrontend/dont_list.html

```diff
@@ -1,5 +1,11 @@
 <div class="nhsuk-do-dont-list">
-  <h{{ heading_level }} class="nhsuk-do-dont-list__label">Don&#39;t</h{{ heading_level }}>
```

Comment:
I'm not sure why we're using `&#39;` here. Should be using the unicode "RIGHT SINGLE QUOTATION MARK" (’) https://www.fileformat.info/info/unicode/char/2019/index.htm

Reply:
Yeah, that is ugly – I've fixed that.